Fix hardcoding of /usr/bin/perl, use perl available through $PATH
author: Nathan S. Watson-Haigh (ACPFG)
--- fastqc.orig 2016-03-01 10:53:17.433849105 +1030
+++ fastqc      2016-03-01 10:53:44.850579745 +1030
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 use warnings;
 use strict;
 use FindBin qw($RealBin);

